modules/ztp-add-local-reg-for-sno-duprofile.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: CONCEPT
+[id="ztp-add-local-reg-for-sno-duprofile_{context}"]
+= Configuring the Image Registry Operator for local caching of images
+
+{product-title} manages image caching with a local registry. A Single Node OpenShift (SNO) Distributed Unit (DU) is subject to bandwidth restrictions when communicating with the centralized registry. This can lead to long image download times.
+
+This is unavoidable for the initial deployment. Over time, there is a risk that `crio` will wipe `/var/lib/containers/storage` in the case of a disorderly shutdown.
```

Comment:
As the start of a new paragraph, let's define what "this" is: 

"Long download times are unavoidable during initial deployment."

Similarly, avoid using "wipe" - let's stick with industry standard terms, like "erase". Disorderly is fine, but in Linux we typically refer to it as an "unexpected shutdown": 
"Over time, there is a risk that `crio` wil erase the `/var/lib/containers/storage` (directory, file?) in case of an unexpected shutdown."